app/controllers/provider_interface/provider_interface_controller.rb

```diff
@@ -40,16 +40,6 @@ def dfe_sign_in_user
     def authenticate_provider_user!
       return if current_provider_user
```

Comment:
Should `current_provider_user` and `current_support_user` be memoized?

Reply:
Might as well, the controllers inheriting from these controllers could choose to do crazy things with these methods!